plugins/Eq/DBvModel.cpp

```diff
@@ -0,0 +1,14 @@
+#include "DBvModel.h"
+
+DBvModel::DBvModel(float val, float min, float max, float step,
+								   Model *parent, const QString &displayName,
+								   bool defaultConstructed) :
+		FloatModel( val, min, max, step, parent, displayName, defaultConstructed )
+{
+		connect(this, SIGNAL( dataChanged() ), this,SLOT( calcAmp() ) );
```

Comment:
Bad idea to use signals for this. Causes more overhead and possible latency. You could look at how I've done the dBV to linear conversion in CrossoverEq: by using the isValueChanged() method, you can remove the need to use signals.